scripts/intl/intl-zanata-pull.sh

```diff
@@ -1,9 +1,10 @@
 #! /usr/bin/env bash
 [[ -e package.json ]] || cd ../..
 
-JAVA_HOME=$(alternatives --list | grep 'jre_1.8.0[[:space:]]' | cut -f 3)
+JAVA_HOME=$(alternatives --list | grep 'jre_1.8.0' | head -1 | cut -f 3)
+ZANATA_LOCALES=de,es,fr,it,ja,ko,pt-BR,zh-CN,cs
```

Comment:
It would be nice to have the list of supported locales in one place and use it both for Zanata pull/push and in the UI (`BASE_LOCALE_SET` in `src/intl/index.js`). 
@sjd78  What do you think? (of course new PR)

Reply:
Could be done sure.  The locales would need to be in a json file and the bash script would probably then use `jq` to generate the list for the command.  Followup PR for sure.  Feel free to give it a try if you like!